### ISSUE TITLE
TreeGrafter issue PANTHER:PTHR10333:SF42

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5061]
UniProtKB:O74736 | png2 | enables | GO:0004402    histone acetyltransferase activity | ECO:0007826   IEA | GO_REF:0000118 | PANTHER:PTHR10333:SF42 | 284812 Schizosaccharomyces pombe (strain 972 / ATCC 24843) | TreeGrafter

Png2 is a  histone acetyltransferase  subunit, but it doesn't have histone acetyltransferase activity.

Need to track down/report  originating annotation.

**Comment by ValWood** [comment]
@pgaudet this is an example with no IBA (there are lots)

**Comment by ValWood** [comment]
We don't propagate these though do we? At least I have never seen it.

**Comment by pgaudet** [comment]
In this case it would be a question for @alexsign - do these get propagated in the GOA pipeline? 

At GO central TBH I am not sure what we do. On the list...

**Comment by ValWood** [comment]
It's odd because if they were propagated, I would have 1000s for complex members. Maybe it's based on an annotation that has been subsequently removed....

**Comment by pgaudet** [comment]
@alexsign says that interontology links would NOT come from TreeGrafter as the source, so my hypothesis was incorrect.

**Comment by pgaudet** [comment]
@dustine32 It is possible to check in the PAINT database whether UniProtKB:O74736  ever had an annotation to GO:0004402    histone acetyltransferase activity ?

**Comment by dustine32** [comment]
@pgaudet No GO:0004402 in the PAINT Curation DB. I only see these terms annotated to UniProtKB:O74736:
```
GO:0035064	IBA
GO:0005829	HDA
GO:0033698	IBA
GO:0033698	IDA
GO:0005634	IBA
GO:0005634	HDA
GO:0006355	IBA
GO:0140861	IC
```

**Comment by pgaudet** [comment]
Is this current? Or does that cover all the history?

**Comment by dustine32** [comment]
@pgaudet Yes, this is current and covers all history. There doesn't appear to be any previously obsoleted GO annotation to this protein.
